modules/gltf/gltf_document.cpp

```diff
@@ -4213,6 +4213,35 @@ Error GLTFDocument::_parse_materials(Ref<GLTFState> p_state) {
 			}
 		}
 
+		// Supersedes KHR_materials_pbrSpecularGlossiness
+		// see https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_specular/README.md
+		if (material_extensions.has("KHR_materials_specular")) {
+			Dictionary specular_properties = material_extensions["KHR_materials_specular"];
+			if (specular_properties.has("specularFactor")) {
+				float specular_value = specular_properties["specularFactor"];
+				material->set_specular(specular_value / 2.0);
+
+			} else if (specular_properties.has("specularColorFactor")) {
+				const Array &arr = specular_properties["specularColorFactor"];
+				ERR_FAIL_COND_V(arr.size() != 3, ERR_PARSE_ERROR);
+				float specular_value = arr[0];
+				if (specular_value > 2.0) {
+					specular_value = 2.0;
+				}
+				// we don't have support for specular tint color, but there's no way
+				// of knowing just the specular factor if the tint color is not (1,1,1)
+				// Therefore, enforce the constraint that all values must be equal
+				// However we can't know if (0.5, 0.5, 0.5) is a gray tint * specular factor of 1
+				// or a specular factor of 0.5. That's up to the user to set the Blender model
+				// correctly for Godot usage
```

Comment:
Instead of failing if the model has a specular tint, we should do our best to approximate. Is it best to use the red channel, or would it be better to use `Color::get_v` to get the HSV value?

Let's simplify this comment, and change it so that the `if ((arr[0] == arr[1]) && (arr[1] == arr[2])) {` condition only controls when the warning is shown, but even if the warning is shown, we should still use the specular value.

```suggestion
				// Godot does not support specular tint color, just a single specular factor.
				// Extract a single value from the tint color and use it as the specular factor.
```